pkg/stub/handler.go

```diff
@@ -588,6 +588,10 @@ func (h *Handler) Handle(event util.Event) error {
 		// Every time we see a change to the Config object, update the ClusterOperator status
 		// based on the current conditions of the Config.
 		cfg = h.refetchCfgMinimizeConflicts(cfg)
+		//TODO remove this setting of version once we start getting samples for z or ppc
+		if len(cfg.Spec.Architectures) > 0 && cfg.Spec.Architectures[0] != v1.AMDArchitecture && cfg.Spec.Architectures[0] != v1.X86Architecture {
+			cfg.Status.Version = h.version
```

Comment:
Shouldn't we _always_ set this, and vary whether or not it is reported based on the `Available` status?

Reply:
no we shouldn't ... all that was sorted out in 4.0/4.1 initial dev

only report the version once you have achieved it